test/unit/factorization/mc/test_compute_t_factor.cpp

```diff
@@ -275,7 +385,7 @@ TYPED_TEST(ComputeTFactorDistributedTest, Correctness) {
       // and that TypeUtilities<TypeParam>::error indicates maximum error for a multiplication + addition.
       SCOPED_TRACE("Comparison test");
       const auto error =
-          h_result.size().rows() * t.size().rows() * test::TypeUtilities<TypeParam>::error;
+          h_result.size().rows() * h_expected.size().rows() * test::TypeUtilities<TypeParam>::error;
```

Comment:
Why do you increase the error threshold?
Note: `t.size().rows() = k`, while `h_expected.size().rows() = h_result.size().rows() = m`